template/data/fr/pages.json

```
@@ -0,0 +1,308 @@
+          "submitButton": {
+            "__component": "links.button",
+            "text": "S'Inscrire",
+            "type": "primary"
+          },
+          "location": "Home Page Bottom"
+        },
+        {
+          "__component": "sections.bottom-actions",
+          "title": "Start building your website",
+          "buttons": [
+            {
+              "url": "https://github.com/strapi/strapi-template-corporate",
+              "newTab": false,
+              "text": "Commencer",
+              "type": "primary"
+            },
+            {
+              "url": "https://github.com/strapi/strapi-template-corporate",
+              "newTab": false,
+              "text": "Voir le code",
+              "type": "secondary"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "id": 6,
+      "slug": "pricing-fr",
+      "locale": "fr",
+      "localizations": [
+        {
+          "id": 2,
+          "locale": "en"
+        }
+      ],
+      "shortName": "Pricing",
+      "status": "published",
+      "metadata": {
+        "metaTitle": "Pricing",
+        "metaDescription": "Les plans différents",
+        "twitterCardType": "summary",
+        "twitterUsername": null,
+        "shareImage": null
+      },
+      "contentSections": [
+        {
+          "__component": "sections.pricing",
+          "title": "Tableaux de tarifs",
+          "plans": [
+            {
+              "name": "Hobby",
+              "description": "Parfait pour des projets personnel",
+              "isRecommended": null,
+              "price": 0,
+              "pricePeriod": "forever",
+              "features": [
+                {
+                  "name": "Voilà une feature"
+                },
+                {
+                  "name": "Une autre"
+                },
+                {
+                  "name": "Encore une autre"
+                }
+              ]
+            },
+            {
+              "name": "Premium",
+              "description": "Un plan plus avancé",
+              "isRecommended": true,
+              "price": 20,
+              "pricePeriod": "per month",
+              "features": [
+                {
+                  "name": "Feature 1"
+                },
+                {
+                  "name": "Feature 2"
+                },
+                {
+                  "name": "Feature 3"
+                }
+              ]
+            },
+            {
+              "name": "Entreprise",
+              "description": "Pour les besoin des grands entreprises",
+              "isRecommended": null,
+              "price": 299,
+              "pricePeriod": "per month",
+              "features": [
+                {
+                  "name": "Amazing feature"
+                },
+                {
+                  "name": "Wow effect feature"
+                },
+                {
+                  "name": "Mesmerizing feature"
+                }
```

Let's keep it in french even if it's not real content

```suggestion
                {
                  "name": "Fonctionnalité 1"
                },
                {
                  "name": "Fonctionnalité 2"
                },
                {
                  "name": "Fonctionnalité 3"
                }
```